ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3601,6 +3601,21 @@ AHU_DFSS_DXZC_ECONZ_EFSS2X_HT2ZC_CO2C:
   - HT2ZC
   - CO2C
 
+AHU_DFSS_DXRC_RHDHC_AL:
```

Comment:
does not contain return or outside air control, required for AHUs.

Reply:
Yes i think its my mistake to add it in AHU adding it in FCU. Does that works?